datafusion/sqllogictest/test_files/select.slt

```diff
@@ -46,11 +46,300 @@ STORED AS CSV
 WITH HEADER ROW
 LOCATION '../core/tests/data/aggregate_simple.csv'
 
-
 ##########
 ## SELECT Tests
 ##########
 
+##########
+## window_null_string_table_test
+##########
+
+statement ok
+CREATE TABLE window_null_string_value_prepare(x string, y string, z string)
+AS VALUES
+('one', 'ONE', 'One'),
+(NULL, NULL, NULL),
+('three', 'THREE', 'Three');
+
+statement ok
+CREATE TABLE window_null_string_table
+AS SELECT arrow_cast(x, 'Dictionary(Int32, Utf8)') as d1,
+y as d2,
+arrow_cast(z, 'LargeUtf8') as d3 FROM window_null_string_value_prepare;
+
+query ?I
+SELECT d1, row_number() OVER (partition by d1) as rn1 FROM window_null_string_table order by d1 asc;
+----
+one 1
+three 1
+NULL 1
+
+query TI
+SELECT d2, row_number() OVER (partition by d2) as rn1 FROM window_null_string_table ORDER BY d2 asc;
+----
+ONE 1
+THREE 1
+NULL 1
+
+query TI
+SELECT d2, row_number() OVER (partition by d2 order by d2 desc) as rn1 FROM window_null_string_table ORDER BY d2 desc
+----
+NULL 1
+THREE 1
+ONE 1
+
+##########
+## query_parameters_table_test
+##########
+
+statement ok
+CREATE TABLE query_parameters_table (
+  c1 BIGINT NOT NULL,
+  c2 BIGINT NOT NULL,
+  c3 BOOLEAN NOT NULL,
+) AS VALUES (0, 1, false),
+(0, 10, true),
+(0, 2, true),
+(0, 3, false),
+(0, 4, true),
+(0, 5, false),
+(0, 6, true),
+(0, 7, false),
+(0, 8, true),
+(0, 9, false),
+(1, 1, false),
+(1, 10, true),
+(1, 2, true),
+(1, 3, false),
+(1, 4, true),
+(1, 5, false),
+(1, 6, true),
+(1, 7, false),
+(1, 8, true),
+(1, 9, false),
+(2, 1, false),
+(2, 10, true),
+(2, 2, true),
+(2, 3, false),
+(2, 4, true),
+(2, 5, false),
+(2, 6, true),
+(2, 7, false),
+(2, 8, true),
+(2, 9, false),
+(3, 1, false),
+(3, 10, true),
+(3, 2, true),
+(3, 3, false),
+(3, 4, true),
+(3, 5, false),
+(3, 6, true),
+(3, 7, false),
+(3, 8, true),
+(3, 9, false);
+
+# test_list_query_parameters
+query IIB
+SELECT * FROM query_parameters_table WHERE c1 = 3;
```

Comment:
I think this is different than the original test where the parameter is filled in via `with_param_value`. I will restore it back to the .rs file

```
        .sql("SELECT * FROM test WHERE c1 = $1")
```